src/DebugTrait.php

```diff
@@ -131,8 +128,6 @@ public function debugTraceChange(string $trace = 'default'): void
      * System is unusable.
      *
      * @param string $message
-     *
-     * @return void
      */
     public function emergency($message, array $context = [])
```

Comment:
Why not moving type `string $message`  to method parameters instead of phpdoc comment?
In other places you do like that.
Same goes for all these similar methods below.

Reply:
Because of  parent interface :)